.github/CHANGELOG.md

```diff
@@ -536,49 +520,35 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
 * Added custom gate application for Toffoli in `default.qubit`.
   [(#1249)](https://github.com/PennyLaneAI/pennylane/pull/1249)
 
+* Added validation for noise channel parameters. Invalid noise parameters now
+  raise a `ValueError`.
+  [(#1357)](https://github.com/PennyLaneAI/pennylane/pull/1357)
+
 * The device test suite now provides test cases for checking gates by comparing
   expectation values.
   [(#1212)](https://github.com/PennyLaneAI/pennylane/pull/1212)
 
 * PennyLane's test suite is now code-formatted using `black -l 100`.
   [(#1222)](https://github.com/PennyLaneAI/pennylane/pull/1222)
 
-* Adds a `hamiltonian_expand` tape transform. This takes a tape ending in
-  `qml.expval(H)`, where `H` is a Hamiltonian, and maps it to a collection
-  of tapes which can be executed and passed into a post-processing function yielding
-  the expectation value.
-  [(#1142)](https://github.com/PennyLaneAI/pennylane/pull/1142)
-
-  Example use:
-
-  ```python
-  H = qml.PauliZ(0) + 3 * qml.PauliZ(0) @ qml.PauliX(1)
-
-  with qml.tape.QuantumTape() as tape:
-      qml.Hadamard(wires=1)
-      return qml.expval(H)
-
-  tapes, fn = qml.transforms.hamiltonian_expand(tape)
-  dev = qml.device("default.qubit", wires=3)
-  res = dev.batch_execute(tapes)
-  ```
-  ```pycon
-  >>> fn(res)
-  4.0
-  ```
-
 * PennyLane's `qchem` package and tests are now code-formatted using `black -l 100`.
   [(#1311)](https://github.com/PennyLaneAI/pennylane/pull/1311)
 
 <h3>Breaking changes</h3>
 
+* The `qml.inv()` function is now deprecated with a warning to use the more general `qml.adjoint()`.
+  [(#1325)](https://github.com/PennyLaneAI/pennylane/pull/1325)
+
 * Removes support for Python 3.6 and begin testing for Python 3.9.
   [(#1228)](https://github.com/XanaduAI/pennylane/pull/1228)
 
```

Comment:
Even if it is duplicated in another section, best to briefly state _all_ deprecations here, so that they are easily referenced. One that comes to mind is `tape.get_resources` and `tape.get_depth` (mentioned in the `specs` entry). Are there any others?

Reply:
> One that comes to mind is tape.get_resources and tape.get_depth (mentioned in the specs entry).

Just double-checking: so we'd like to add "ongoing" deprecations too to the "Breaking changes" section, correct? As this one is not yet deprecated, but will be.

I've updated this section to have the ongoing deprecations too. One other, that came to mind was using `qml.sample` with a device that has `shots=None`, where we'll like to turn to an error instead of a warning.

Reply:
Fantastic! Thanks Antal